src/main/kotlin/eli/inevitable/Inevitable.kt

```diff
@@ -0,0 +1,156 @@
+package eli.inevitable
+
+import kotlin.text.RegexOption.DOT_MATCHES_ALL
+
+/**
+ * System agnostic character sequence for generating a line break.
+ */
+val LINE_BREAK = System.lineSeparator()
+/**
+ * Regex pattern for identifying and breaking up new lines.
+ */
+val LINE_BREAK_PATTERN = Regex("""(\r\n|\r|\n)""", DOT_MATCHES_ALL)
+
+/**
+ * Regex pattern for identifying and replacing tab characters.
+ */
+val TAB_PATTERN = Regex("""\t""", DOT_MATCHES_ALL)
+
+/**
+ * Regex pattern for identifying all non-printable characters (excluding space ' ').
+ */
+val NON_DISPLAY_PATTERN = Regex("""^[\S ]+$""", DOT_MATCHES_ALL)
+
+/**
+ * Extension function to determine the max width of a [collection of strings][Collection]
+ */
+fun Collection<String>.maxWidth(): Int = this.maxBy { it.length }?.length ?: 0
+
+/**
+ * Breaks apart a single [string of text][String] into multiple lines if there are any line
```

Comment:
I feel like 'string of text' is redundant here, since a string has chars and it is text. I guess it could be a string of emojis...... but "apart a single [String]" feels good enough to me.

Reply:
Valid point. I'll rephrase